content/advanced/340_appmesh_flagger/canary_deploy.md

```diff
@@ -0,0 +1,201 @@
+---
+title: "Deploy Canary Set Up"
+date: 2021-03-18T00:00:00-05:00
+weight: 30
+draft: false
+---
+
+#### Clone the repository
+
+```
+cd ~/environment
+git clone https://github.com/aws-containers/eks-microservice-demo.git
+cd eks-microservice-demo
+```
+
+#### Build and Push the container images
```

Comment:
I think we can fix this sub title to "Build and Push container images"

Reply:
done